app/models/user.rb

```diff
@@ -6,6 +6,7 @@ class User < ApplicationRecord
   has_many :friends, through: :friendships
   has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
   has_many :inverse_friends, through: :inverse_friendships, source: :user
+  has_many :confirmed_friendships, -> { where confirmed: true }, class_name: 'Friendship'
```

Comment:
- [ ] There is a duplicate association for `inverse_friendships` here.Let's remove the first one.

Reply:
We are using it if we take it out then we have errors.